test/transform/test_transform.py

```diff
@@ -1046,13 +1046,10 @@ def make_dataset(N, train_length):
 def assert_serializable(x: transform.Transformation):
     t = fqname_for(x.__class__)
     y = load_json(dump_json(x))
-    z = load_code(dump_code(x))
+
     assert dump_json(x) == dump_json(
         y
     ), f"Code serialization for transformer {t} does not work"
-    assert dump_code(x) == dump_code(
-        z
-    ), f"JSON serialization for transformer {t} does not work"
```

Comment:
Assertion messages here appeared to be swapped?

Reply:
Not sure I understand